lib/formtools/plugins/plugins-helpers.js

```diff
@@ -390,6 +383,12 @@ var formSettings = function (schema, form, labels) {
     for (var key in formConfigs){
         schemaConfig = formConfigs[key] || {};
         formConfig = form[key] || {};
+
+        // support for form DSL where "fieldName: true" with no other configurations
+        if (typeof formConfig === 'boolean' && formConfig === true) {
```

Comment:
You could remove the `typeof formConfig === 'boolean' &&` part, tripe equals will suffice.